upgrade-notes/backend_20251205_132146.md

```diff
@@ -0,0 +1,5 @@
+#### Fixed GrapesJS editor content corruption ([#4313](https://github.com/shopsys/shopsys/pull/4313))
+
+- fixed `EnsureCorrectGrapesJsFormatHelper` to properly detect existing GrapesJS components and prevent wrapping already structured content into nested `gjs-text-ckeditor` divs
+- this resolves the issue where adding an image before text in articles/blog posts caused the editor to break after saving and reopening
+- see #project-base-diff to update your project
```

Comment:
There is no change in the project-base. 

I think we can remove upgrade notes altogether